### PR TITLE
Add link to logout URL

### PR DESCRIPTION
**Pull request by garyttierney**

### CODE REVIEW

File: index.css [hunk 1]
```diff
@@ -66,4 +66,25 @@ body {
...
+
+#uv-container {
+    position: relative;
+}
+
+#logout {
+    top: 0;
+    right: 0;
+    position: absolute;
+    padding: 10px;
```
Comment (stephenwf) [inline index.css]: We could move the padding to the `a` tag to give that extra bit of room for the link to register

File: index.css [hunk 2]
```diff
@@ -66,4 +66,25 @@ body {
...
+}
+
+#logout a {
+    font-size: 12px;
+    font-weight: normal;
+    line-height: 1.42857143;
```
Comment (stephenwf) [inline index.css:88]: that's 20/14 right?

File: index.css (hunk 1 above)
Comment (garyttierney) [reply index.css]: I tried this but then the padding seemed not to take effect.

File: index.css (hunk 2 above)
Comment (garyttierney) [reply index.css:88]: Yeah, I copied it from one of the existing links they've styled.

File: index.css (hunk 1 above)
Comment (stephenwf) [reply index.css]: `display: inline-block` should do the trick on the a tag

**Review by danielgrant — APPROVED** [review]

**Pull request merged**